Salesforce/trigger_salesforce_events.json

```diff
@@ -22,6 +22,11 @@
         "type": "integer",
         "description": "Batch frequency in seconds",
         "default": 600
+      },
```

Comment:
Could you update [connector_salesforce_events.json](https://github.com/SEKOIA-IO/automation-library/blob/main/Salesforce/connector_salesforce_events.json) also?

Reply:
Along with `fetch_daily_logs` I also added `frequency` to `connector.json` to make them sync. It is fine?